numpy/linalg/tests/test_linalg.py

```diff
@@ -61,30 +61,44 @@ def get_rtol(dtype):
 
 class LinalgCase(object):
 
-    def __init__(self, name, a, b, exception_cls=None):
+    def __init__(self, name, a, b, flags=frozenset()):
```

Comment:
The class should be documented, at least for the constructor. This is especially so now with the rather generic `flags`.